### PR TITLE
adding the ami section per SG-23248

### PR DESCRIPTION
**Pull request by shaynacohen**

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/d4667e81a24246b1f1580b483b49a6db797bd7b1/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/e1586308545bf0e1876af11694b420e2380509f4/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/692efda36881f4292989d21e1b2447adf7a3b2bb/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/d0b47ef3ff0340331a688dde5908afac1221da41/index.html)

**Comment by shaynacohen** [comment]
Note: links within the doc will need to be updated after the merge so that I can get the appropriate links.

File: docs/en/action-menu-items/action-menu-items-create.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+*   User brings up the context menu (by right-clicking or clicking on the gear menu in the toolbar)
+*   User clicks ‘Build PDF Report’
+*   {% include product %} dispatches a POST request to the AMI’s URL in a new window (If your {% include product %} server is served over HTTPS, and your URL is served over HTTP, expect to see a browser warning)
+*   Your script, located at the URL specified, processes the POST data, and generates a PDF document
+*   A nicely formatted PDF report is sent back to the user for viewing or download
+
+## Custom protocol handler example
+
+A more advanced implementation of custom AMIs involves setting up a custom protocol handler (e.g., shotgun://process_version). This allows you to bridge {% include product %} with applications like Maya, RV, or Cinesync via a script on your local computer. When you specify a non HTTP(S) protocol, data is sent via GET instead of POST to your URL. You may want to use this to launch an in-house tool to dispatch different requests.
+
```
Comment (000paradox000) [inline docs/en/action-menu-items/action-menu-items-create.md:52]: Not sure if the word shotgun is right in this example

File: docs/en/action-menu-items/custom-browser-protocols.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+```
+[HKEY_CLASSES_ROOT\shotgun]
+@="URL:shotgun Protocol"
+"URL Protocol"=""
+[HKEY_CLASSES_ROOT\shotgun\shell]
+[HKEY_CLASSES_ROOT\shotgun\shell\open]
+[HKEY_CLASSES_ROOT\shotgun\shell\open\command]
+@="python""sgTriggerScript.py""%1"
+```
```
Comment (000paradox000) [inline docs/en/action-menu-items/custom-browser-protocols.md:84]: Not sure if thw windows registry key will be shotgun or shotgrid

File: docs/en/action-menu-items/custom-browser-protocols.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+    </array>
+```
+
+You may want to change the following three strings:
+```
+com.mycompany.AppleScript.Shotgun
+Shotgun
+shotgun
+```
+
```
Comment (000paradox000) [inline docs/en/action-menu-items/custom-browser-protocols.md:204]: Not sure if Shotgun is correct or maybe Shotgrid

**Review by 000paradox000 — APPROVED** [review]
LGTM

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/e02e9ada79923dc359b513420df034e87c727f7e/index.html)

**Pull request merged**